entity-service/src/main/kotlin/com/egm/stellio/entity/model/EntityAccessControl.kt

```diff
@@ -0,0 +1,82 @@
+package com.egm.stellio.entity.model
+
+import com.egm.stellio.entity.config.Neo4jUriPropertyConverter
+import com.egm.stellio.shared.util.JsonLdUtils
+import com.egm.stellio.shared.util.toNgsiLdFormat
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.springframework.data.annotation.LastModifiedDate
+import org.springframework.data.neo4j.core.convert.ConvertWith
+import org.springframework.data.neo4j.core.schema.DynamicLabels
+import org.springframework.data.neo4j.core.schema.Id
+import org.springframework.data.neo4j.core.schema.Node
+import java.net.URI
+import java.time.Instant
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+
+@Node
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class EntityAccessControl(
+
+    @Id
+    @JsonProperty("@id")
+    @ConvertWith(converter = Neo4jUriPropertyConverter::class)
+    val id: URI,
+
+    @DynamicLabels
+    @JsonProperty("@type")
+    val type: List<String>,
+
+    @JsonIgnore
+    val createdAt: ZonedDateTime = Instant.now().atZone(ZoneOffset.UTC),
```

Comment:
setting a default value here is not what you really want

Reply:
Yes but I have to retrieve this information during the request then ?